assets/js/admin/customizer.js

```diff
@@ -0,0 +1,487 @@
+/* global _wpCustomizeWcApiDevGuidedTourSteps, _wpCustomizeWcApiDevGuidedSettings */
+( function( wp, $ ) {
+	'use strict';
+
+	if ( ! wp || ! wp.customize ) { return; }
+
+	// Set up our namespace.
+	var api = wp.customize;
+	var settings = _wpCustomizeWcApiDevGuidedSettings || {};
+
+	api.WcApiDevGuidedTourSteps = [];
+
+	if ( 'undefined' !== typeof _wpCustomizeWcApiDevGuidedTourSteps ) {
+		$.extend( api.WcApiDevGuidedTourSteps, _wpCustomizeWcApiDevGuidedTourSteps );
+	}
+
+	/**
+	 * wp.customize.WcApiDevGuidedTour
+	 *
+	 */
+	api.WcApiDevGuidedTour = {
```

Comment:
Like the query string, I would probably go with another name here and leave `wc-api-dev` out of the name. During Q4 we are going to split out all none-API logic from `wc-api-dev` into a separate always loaded plugin for Stores.